src/server/auth/router.js

```diff
@@ -0,0 +1,79 @@
+import express from 'express';
+import controller from './controllers';
+
+const router = express.Router();
+
+/**
+ * @api  /auth/refresh refresh token
+ * @apiName refreshToken
+ * @apiGroup Auth
+ *
+ * @apiParam {String} refreshToken
+ * @apiParamExample {json} Request-Example:
+ *      {
+ *          "refreshToken": "refreshToken"
+ *      }
+ *
+ * @apiSuccess {String} accessToken new access token
+ * @apiSuccess {String} refreshToken new refresh token
+ *
+ * @apiSuccessExample Success-Response:
+ *     HTTP/1.1 200 OK
+ *     {
+ *       "accessToken"  : "new access token",
+ *       "refreshToken" : "new refresh token"
+ *     }
+ *
+ * @apiError (Error 400) error the refresh token is invalid.
+ * @apiErrorExample {json} Invalid-Refresh-Token-Response:
+ *     HTTP/1.1 400 Bad Request
+ *     {
+ *       error: "error message ..."
+ *     }
+ *
+ */
+router.post('/auth/refresh', controller.refreshToken);
```

Comment:
I think the address has to be changed 
Please check out server/router/api-routes.js & server/<module>/router.js
OR check out server/index.js